src/pages/term/nextBackButtons/nextBackButtons.tsx

```diff
@@ -0,0 +1,16 @@
+import { NextButton, BackButton } from '~components/UI/Buttons/Buttons';
+import styles from "./nextBackButtons.module.sass";
+
+interface NextBackButtonsProps {
+  visitNextItem: () => void;
+  visitPreviousItem: () => void;
```

Comment:
They should optional, in case there isn't next / previous item.
Also, the right term for events is `onClick`, so the name should be `onNextItemClick`.
```suggestion
  onNextItemClick?: () => void;
  onPreviousItemClick?: () => void;
```